schema.d.ts

```diff
@@ -45,7 +45,7 @@ export interface Package {
    * Whether the package is deprecated.
    * If the value is a string, it's the reason for the deprecation.
    */
-  deprecated?: boolean|string;
+  deprecated?: boolean | string;
```

Comment:
huh, why are we checking in the .d.ts file?

oh, this is hand written as an expression of the json schema, it's not an output from the typescript compiler?

Reply:
yeah, this is the source of truth.